src/zh/deploy/config.md

```diff
@@ -68,6 +68,19 @@ RSSHub 支持使用访问密钥 / 码进行访问控制。开启将会激活全
 
 -   或使用访问密钥 `key` 直接访问所有路由，例如：`https://rsshub.app/qdaily/column/59?key=ILoveRSSHub`
 
+**Healthcheck 配置**
+
+当启用 `ACCESS_KEY` 时，`healthcheck` 端点也需要进行身份验证。
+
+对于 Docker Compose 部署，你需要在 `docker-compose.yml` 中更新 `healthcheck` 配置以包含访问密钥或访问码参数。
```

Comment:
```suggestion
对于 Docker Compose 部署，你需要在 `docker-compose.yml` 中更新 `healthcheck` 配置以包含访问密钥或访问码参数。

```

Reply:
Do you mean that I should add one more blank line here?

However, doing so would result in two consecutive blank lines before the next code block, which would violate markdownlint [MD012](https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md012.md):

```
MD012/no-multiple-blanks: Multiple consecutive blank lines [Expected: 1; Actual: 2]
```

Reply:
BTW, I don't know if there's a unified Markdown format specification currently.
So I didn't fix all the markdownlint warnings in this modified document.

Reply:
I don't think changing from
```
include the access key or access code parameter.
The recommended configuration is:
```
to
```
include the access key or access code parameter.

The recommended configuration is:
```
violates MD012.

Reply:
Well, I didn't see clearly.